samples/samples.js

```diff
@@ -105,7 +105,7 @@
 			path: 'scales/logarithmic/scatter.html'
 		}]
 	}, {
-		title: 'Time scale',
+		title: 'Time and TimeSeries scale',
```

Comment:
Can we keep `title: 'Time scales'` and avoid the title on 2 lines?

![image](https://user-images.githubusercontent.com/3874900/28008456-e55263b4-6557-11e7-97e0-77c0a9631c16.png)